frontend/src/pages/ModelTester/ControlButtons2/ControlButtons2.jsx

```diff
@@ -54,6 +53,12 @@ const ControlButton2 = ({ setUploadedFiles }) => {
 
     useEffect(() => {
         fetchEmailAndDemographics();
+
+        // Load the uploaded files from localStorage when the component mounts
+        const storedFiles = JSON.parse(localStorage.getItem("uploadedFiles"));
```

Comment:
Why exactly are we using local storage here

Reply:
When the user reloaded the page, the uploaded models would go. We want the models to still show so that the user doesn't have to upload them again